app/services/course/assessment/question/programming/python/python_package_service.rb

```diff
@@ -151,7 +151,14 @@ def generate_zip_file(data_files_to_keep)
       index = 1
       [:public, :private, :evaluation].each do |test_type|
         tests[test_type].try(:each) do |test|
-          test_fn = "    def test_#{test_type}_#{index}(self):\n"\
+
+          # Hard limit of 99 test cases so we only need to prepend
+          # 1 '0' to single index test cases
+          if index > 99 then
```

Comment:
This part of the code is removed in the edit because it fails silently (e.g. the test cases are present in the database but it is not created during testing).